components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceConstants.java

```diff
@@ -80,6 +80,11 @@
     public static final String ENTRY_VALUE_SEPARATOR = ":";
 
     /**
+     * This is the character used in lifecycle property keys
+     */
+    public static final String DOT = ".";
```

Comment:
This should be a meaningful name that describe the purpose of this variable (something like "lifecycle property separator")  not a name to describe the value.